src/_pytest/logging.py

```diff
@@ -253,6 +253,13 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=None,
         help="log date format as used by the logging module.",
     )
+    add_option_ini(
+        "--log-cli-level-color",
+        dest="log_cli_level_color",
+        default="yes",
```

Comment:
I've just seen that you used True/"bool" in https://github.com/pytest-dev/pytest/pull/8305/commits/c90f2aca312944c9d375b86ee1dc9aa44a6072e5,  what was the problem with it?

Reply:
Good catch, I switched to `yes/no` to conform to `--code-highlight={yes,no}` but since I can remove the cli flag it would make more sense to share the same convention than `log_cli` indeed